index.html

```diff
@@ -169,36 +169,36 @@ <h3 class="subheader">PRIZES:</h3>
                 </div>
                 <div class="row eventBox">
                   <div class="col eventBoxHeading">
-                      <h3 class="subheader">SEPTEMBER 23</h3>
+                      <h3 class="">September 23</h3>
                       <h2>Community Livestream</h2>
                       <span class="eventTime">00:00PM EST</span>
                   </div>
-                  <div class="col"></div>
-                  <div class="col">
+                  <div class="col2">
                       <a href="" class="btn btn-outline">Learn More</a>
                   </div>
                 </div>
                 <div class="row eventBox">
                   <div class="col eventBoxHeading">
-                      <h3 class="subheader">SEPTEMBER 23</h3>
+                      <h3>September 23</h3>
                       <h2>The State of Open Source featuring DigitalOcean, Hashicorp, and Spotify</h2>
                       <span class="eventTime">00:00PM EST</span>
                   </div>
-                  <div class="col centered">
-                    <h3 class="subheader">special sponsor</h3>
+                  <div class="col">
+                    <h3 class="">Special sponsors</h3>
                     <ul id="sponsorList">
-                        <li><img src="img/DigitalOcean.png" alt="DigitalOcean"></li>
-                        <li><img src="img/HashiCorp.png" alt="HashiCorp"></li>
-                        <li><img src="img/Spotify.png" alt="Spotify"></li>
+                        <li><img src="img/DigitalOcean.png" style="max-width: 165px;" alt="DigitalOcean"</li>
+                        <li><img src="img/HashiCorp.png" style="max-width: 130px;" alt="HashiCorp"></li>
+                        <li><img src="img/Spotify.png" alt="Spotify" style="max-width: 120px;"></li>
```

Comment:
Yeah couldn't figure out how to make all their logos even without getting really specific which each of them.